src/nupic/types/Sdr.hpp

```diff
@@ -31,14 +31,16 @@
 #include <nupic/types/Serializable.hpp>
 #include <nupic/utils/Random.hpp>
 
-using namespace std;
 
 namespace nupic {
+namespace sdr {
```

Comment:
moves to namespace nupic::sdr; 

that is a lot of files changed then

Reply:
this was reverted

Reply:
Ok. For changes with big diffs like this namespace change: a separate PR is a good idea and makes it easier to review.  This is tracked in the issue sdr_reorganization.